app/src/test/java/com/appunite/loudius/ui/login/LoginScreenViewModelTest.kt

```diff
@@ -0,0 +1,122 @@
+package com.appunite.loudius.ui.login
+
+import io.mockk.every
+import io.mockk.mockk
+import org.junit.jupiter.api.Test
+import strikt.api.expectThat
+import strikt.assertions.isEqualTo
+import strikt.assertions.isFalse
+import strikt.assertions.isNull
+import strikt.assertions.isTrue
+
+class LoginScreenViewModelTest {
+
+    private val githubHelper = mockk<GithubHelper> {
+        every { shouldAskForXiaomiIntent() } returns false
+    }
+    private fun create() = LoginScreenViewModel(githubHelper)
```

Comment:
**Suggestion:** You can have a viewModel variable here and not create it in every test - there is no initialization logic at LoginScreenViewModel. 

`    private val viewModel = LoginScreenViewModel(githubHelper)
`

Reply:
We've discusesed this during the call.